src/main/java/net/ccbluex/liquidbounce/ui/client/hud/element/elements/Text.kt

```diff
@@ -90,29 +97,35 @@ class Text(x: Double = 10.0, y: Double = 10.0, scale: Float = 1F, side: Side = S
         }
 
     private var color: Color
-        get() = Color(red, green, blue)
+        get() = Color(red, green, blue, alpha)
         set(value) {
             red = value.red
             green = value.green
             blue = value.blue
+            alpha = value.alpha
         }
 
-    private fun getReplacement(str: String): String? {
+    private fun getReplacement(str: String): Any? {
```

Comment:
it should be "String?", as it was, you better just change all new arguments type to String?

Reply:
I didn't though, someone else did. https://github.com/CCBlueX/LiquidBounce/blob/legacy/src/main/java/net/ccbluex/liquidbounce/ui/client/hud/element/elements/Text.kt#L108

Reply:
Also, using Any actually makes it easier to add new things, since they don't need to be converted to strings anymore